public/controllers/misc/reporting.js

```diff
@@ -46,6 +47,14 @@ export class ReportingController {
     this.groupToPages();
   }
 
+  offsetTimestamp = time => {
+    try {
+      return this.timeService.offset(time);
```

Comment:
Here is not needed to add text as a parameter like in the previous methods of this PR?

Reply:
No, because in the reporting section is not needed to append a label before the date (e.g: Last scan: )